File: table-cup-frontend/src/app/model/GameModel.ts
export class GameModel {
  uuid: string
  pointsOfTeamBlue: number;
  pointsOfTeamRed: number;
  playersOfBlueTeam: string[];
  playersOfRedTeam: string[];


  constructor(uuid: string, pointsOfTeamBlue: number, pointsOfTeamRed: number, playersOfBlueTeam: string[], playersOfRedTeam: string[]) {
    this.uuid = uuid;
    this.pointsOfTeamBlue = pointsOfTeamBlue;
    this.pointsOfTeamRed = pointsOfTeamRed;
    this.playersOfBlueTeam = playersOfBlueTeam;
    this.playersOfRedTeam = playersOfRedTeam;
  }
}

File: table-cup-backend/src/main/java/ch/teko/tablecupbackend/model/GameModel.java
package ch.teko.tablecupbackend.model;

import java.util.Set;
import java.util.UUID;

public record GameModel(UUID uuid, int pointsOfTeamBlue, int pointsOfTeamRed, Set<String> playersOfBlueTeam,
                        Set<String> playersOfRedTeam) {

}
